code/parachain/frame/assets/src/lib.rs

```diff
@@ -116,6 +118,15 @@ pub mod pallet {
 		#[pallet::constant]
 		type NativeAssetId: Get<Self::AssetId>;
 		type GenerateCurrencyId: CurrencyFactory<AssetId = Self::AssetId, Balance = Self::Balance>;
+		/// Identifier for the class of foreign asset.
+		type ForeignAssetId: FullCodec
```

Comment:
Is this used anywhere in the pallet?

Reply:
Actually no, I added it at some point and then removed it. Well spotted.